django-backend/fecfiler/transactions/managers.py

```diff
@@ -314,7 +314,10 @@ def transaction_view(self):
                 payment_prior=self.PAYMENT_PRIOR_CLAUSE(),
                 payment_amount=self.PAYMENT_AMOUNT_CLAUSE(),
                 loan_key=self.LOAN_KEY_CLAUSE(),
-                loan_payment_to_date=self.LOAN_PAYMENT_CLAUSE(),
+                loan_payment_to_date=Coalesce(
```

Comment:
Adding a default to the Case statement in the clause didn't work because the first When fires when we have the schedule C and does not make it to the default statement because the "if" part is true. I can move the Coalesce up to the LOAN_PAYMENT_CLAUSE so that the transaction_view() stays nice and tidy. See update